engine/src/test/scala/cromwell/engine/backend/WorkflowDescriptorSpec.scala

```diff
@@ -53,7 +53,8 @@ trait WorkflowDescriptorBuilder {
     implicit val timeout = akka.util.Timeout(awaitTimeout)
     implicit val ec = actorSystem.dispatcher
 
-    val actor = actorSystem.actorOf(MaterializeWorkflowDescriptorActor.props(), "MaterializeWorkflowDescriptorActor-" + id.id)
+    val serviceRegistryIgnorer = actorSystem.actorOf(DevNullActor.props)
```

Comment:
I was super scared that we had `DevNullActor` until I noticed the name of this file